core/arch/x86/decode_table.c

```diff
@@ -4017,9 +4017,9 @@ const instr_info_t prefix_extensions[][8] = {
    * SSSE3
```

Comment:
> This is incorrect and should be set to 2 like other pshuf variants.

I would request a rewording of this statement since it sounds like all the pshuf variants are like this one in having the destination also a source, which is not the case.  This is the only pshuf variant where the destination is a source.  (The other ones have an immediate as a 2nd source.)

Reply:
Done